types/protocols.pp

```diff
@@ -0,0 +1,7 @@
+type Etc_services::Protocols = Hash[
+  Enum[
+    'udp',
+    'tcp',
+  ],
+  Integer[1,65535],
```

Comment:
May use the [ `Stdlib::Port` _data type_ ](https://forge.puppet.com/modules/puppetlabs/stdlib/4.25.1/readme#stdlibport) ?

Reply:
Done!